### PR TITLE
Add ability to set country code on mobile log in.

### LINKED ISSUE #192: Mobile log in only supports +1 numbers right now.
This impacts both SSO and account migration.

### PR DESCRIPTION
**Pull request by rahulgi**
Fix #192

### CODE REVIEW

File: src/components/MobileLogIn.tsx [hunk 1]
```diff
@@ -114,24 +119,60 @@ class MobileLogInComponent extends React.Component<Props, State> {
...
-      mobileNumber: formatted
+      countryCode,
+      regionCode,
+      countryCodeError
     });
   };
 
+  private updateMobileNumber = (value: string) => {
+    if (this.state.regionCode) {
+      const formatter = new AsYouTypeFormatter("US");
```
[the remark this answers is not in the document]
Comment (rahulgi) [reply src/components/MobileLogIn.tsx]: derp

**Comment by rahulgi** [comment]
@osdiab what do you mean by that? If they don't add the plus sign it gets added automatically.

oops @redro sorry, haven't deployed yet heheh. once this Pull Request gets approved, I'll land it, deploy, and then close issue #192.

**Comment by redro** [comment]
I’m in!
There was just one issue left. On my iPhone SE the input field was cut off. Only a little piece of it was visible.

**Comment by rahulgi** [comment]
right.. that's a problem. will update.

**Comment by rahulgi** [comment]
https://github.com/rahafoundation/raha-web-app/pull/194

**Pull request merged**